torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3167,6 +3167,55 @@ def sample_inputs_take_along_dim(op_info, device, dtype, requires_grad, **kwargs
             )
 
 
+def error_inputs_aminmax_amax_amin(op_info, device, **kwargs):
+
+    # Error Inputs for zero-dim tensors, when 'dim' arg is not provided.
+    shape = (S, 0, S)
+    err_msg_amax_amin = "Specify the reduction dim with the 'dim' argument."
+    err_msg_aminmax = "cannot compute aminmax over an empty dimension as the operation has no identity"
+    if op_info in ['amax', 'amin']:
```

Comment:
I think this will never be `True` because op_info is object of type `OpInfo` (and this case will never be generated). I think you might want to do `op_info.name`.

Reply:
Yeah 😅